File: Rule_of_Three.py
#Language: python
#Author: Kaywinnet

print(" _____________")
print("|RULE OF THREE|")
print(" ‾‾‾‾‾‾‾‾‾‾‾‾‾")
print("For an equation of the form:")
print("a -------> b")
print("x -------> c")
print("where x is unknown,", end = ' ')
a = float(input("enter a:"))
b = float(input("Now, enter b:"))
c = float(input("Finally, enter c:"))

x = (a*c)/b

print("The value of x is:", x)

File: rule_of_three/rule_of_three.cpp
#include<bits/stdc++.h>

using namespace std;

int main()
{
	cout<<"Rule of three\n";
	cout<<"For an equation of the form :\n";
	cout<<"a ->b\n";
	cout<<"x ->c\n";
	cout<<"where x is unknown\n";
	int a,b,c,x;
	cout<<"Please enter value of a b c ";
	cin>>a>>b>>c;
	x=(a*c)/b;
	cout<<"Value of x is:"<<x;
	return 0;
}

